### DISCUSSION TITLE
Run quarkus continuously tests on a library

### DISCUSSION THREAD

**Discussion by ennishol** [jbangdev/jbang#1635] [Ideas]
Hey jbangers ;)

Would it be possible run a jbang script to launch quarkus continuous testing on a simple maven project with some dependencies (no frameworks)?

**Comment by maxandersen** [comment]
Hi, sorry I missed this. 

Not exactly sure what you are asking for ? Are you asking how you get tests execution when running a jbang app ?

Then currently there isn't a good way to do it as jbang (as of now) don't know about tests. Tests are currently just other jbang apps. 

It should be doable to make quarkus jbang intégration able to pick up tests. Please open issue on quarkus as that's where that would happen.

**Reply by ennishol** [reply]
@maxandersen  Yes it makes more sense in quarkus project. Created https://github.com/quarkusio/quarkus/issues/34898
